ui/raidboss/popup-text.ts

```diff
@@ -1152,9 +1154,8 @@ export class PopupText {
           result = triggerHelper.valueOrFunction(resp.tts);
       }
 
-      result ??= triggerHelper.defaultTTSText;
-
-      triggerHelper.ttsText = result?.toString();
+      if (result)
+        triggerHelper.ttsText = result?.toString();
```

Comment:
Hmm.  What about https://github.com/quisquous/cactbot/search?q=%22tts%3A+null%22?

Or, in other words, how should we say "please don't play tts for this trigger (it is too long / complicated / collides with other tts that is more important)"?

Reply:
I think those triggers are invalid according to the defined data model anyways? Maybe this should be ` | false` to disable tts entirely?

Reply:
I think the data model is still being refined, and even so I think they're valid because they're in `TriggerOutput`.

What about changing `if (result)` to `if (result && result !== undefined)`? That way `undefined` continues to behave as if `tts` wasn't specified at all and `null` and `false` will both prevent tts as they currently do?

Reply:
That would just default it back to the defaultTTSText. I think the block would need to be this instead:

```typescript
      if (result !== undefined) {
        if (result)
          triggerHelper.ttsText = result?.toString();
      } else {
        triggerHelper.ttsText = triggerHelper.defaultTTSText;
      }
```

Reply:
Oh right, thanks for the correction.  Do you think it's worth a "// allow null and false to disable tts" comment to make this more obvious?

Reply:
Adjusted the logic, I think the comment makes sense.